src/main/java/tp/DeleteAppointmentCommand.java

```diff
@@ -17,8 +17,13 @@ public String execute(DoctorList doctorList, PatientList patientList,
                           PatientStorage patientStorage,
                           AppointmentStorage appointmentStorage) throws IHospitalException {
         Appointment curr = appointmentList.deleteAppointment(index);
-        return String.format(boundary + "Noted. I've removed this appointment:\n" + curr
-                                     + "\n" + "Now you have " + (appointmentList.getSize())
-                                     + " appointments recorded in the system." + System.lineSeparator() + boundary);
+        if (curr == null){
+            return String.format("The index " + index + " is not valid in the appointment list");
+        }
+        else {
```

Comment:
Remember to put "else" after the close curly bracket of your "if" statement to follow the consistent style :D